### ISSUE TITLE
Update doc for JavadocMethod

### ISSUE THREAD

**Issue by baratali** [checkstyle/checkstyle#7598]
https://checkstyle.org/config_javadoc.html#JavadocMethod

Add example for each existing config to show what code is ok and what code cause violation.
Good example is https://checkstyle.org/config_naming.html#AbstractClassName
Look at referenced commits of #5832, to see what and how should be changed in code.

Please play with module/check by means of https://checkstyle.org/cmdline.html to understand how it works.

------------

PLEASE READ https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples

**Comment by sudo-andro** [comment]
I am on it

**Comment by ImmortalRabbit** [comment]
I am on it.

**Comment by aryaniiit002** [comment]
i am on it.

**Comment by Devansu-Yadav** [comment]
@ImmortalRabbit Are you still working on this issue? @romani @timurt Can I take up this issue if no PR has been made?

**Comment by Tim-Vuong** [comment]
I am on it (assuming that the last comment was almost a year ago).

**Comment by Tim-Vuong** [comment]
Hello! I've been stuck on mvn verify and mvn clean verify. I have removed all of my changes and realized that my mvn install fails:
<img width="1063" alt="Screen Shot 2022-03-18 at 5 11 58 PM" src="https://user-images.githubusercontent.com/72631688/159083676-fce33342-340e-4a22-ba77-c7b215f604d0.png">

This is the output from mvn clean verify (without my changes):
<img width="1056" alt="Screen Shot 2022-03-18 at 5 13 46 PM" src="https://user-images.githubusercontent.com/72631688/159083852-2541a3dc-c598-4cbe-aad6-d993a9bec379.png">

Other notes: I forked and cloned the repo in VSCode, so I'm not too sure if that could be the reason. However, I tried to use Eclipse and still get warnings/errors in the pom.xml file.

**Comment by nick-mancuso** [comment]
@Tim-Vuong I recommend that you stick to `git` and CLI usage for version control, and use IDE for writing code. Just delete your cloned repo and start fresh.

**Comment by Tim-Vuong** [comment]
@nick-mancuso Thank you for your help! After I restarted, mvn install worked, but I am stuck on passing mvn clean verify.

I added the following in /checkstyle/src/main/checks/javadoc/JavadocMethodCheck.java
<img width="377" alt="Screen Shot 2022-03-19 at 4 02 03 PM" src="https://user-images.githubusercontent.com/72631688/159136666-c5adf13a-4216-4685-bd53-d4166ad3ebba.png">

I added the following in /checkstyle/src/xdocs/config_javadoc.xml
<img width="396" alt="Screen Shot 2022-03-19 at 4 02 46 PM" src="https://user-images.githubusercontent.com/72631688/159136687-6fdb1add-7b7a-4601-9115-7951c628fa9d.png">

This is the error I get from running mvn clean verify:
<img width="781" alt="Screen Shot 2022-03-19 at 4 07 52 PM" src="https://user-images.githubusercontent.com/72631688/159136836-60b2b4c2-4272-488e-b2a8-be9f0b8e9d76.png">

**Comment by NitishKumar06** [comment]
@romani I'm on it.

**Comment by NitishKumar06** [comment]
@romani @rnveach Is this issue fixed? If no, I'd like to work on this issue.

**Comment by romani** [comment]
It is not fixed, please help to fix it

**Comment by NitishKumar06** [comment]
I'm on it

**Comment by rukundob451** [comment]
@romani can I be assigned this issue?

**Comment by rnveach** [comment]
There are no issue assignments. Just say "I am on it" and start working on it.

**Comment by aritrasinha108** [comment]
Since there are no updates here for a month now, should I start working on it?

**Comment by romani** [comment]
@aritrasinha108 , please send PR

**Comment by aritrasinha108** [comment]
@romani one of the tests is failing because of the param tags used as an example. One of the examples I added in ```/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java``` looks something like:
```java
* &#47;**
*  * A sample function to print the product two numbers
*  * @param number1 The first number
*  * @param number1 The second number
*  * @return An integer
*  *&#47;
* public void multiplyTwoNumbers(int number1, int number2){   // ok, param tags available
``` 
when I run ```mvn clean verify``` after making the changes, I get this checkstyle fail 
![Screenshot from 2023-02-28 20-23-11](https://user-images.githubusercontent.com/55765656/221890312-b9c0b300-1a53-45eb-9a6f-e85ee00d8ec6.png)

This happens for all the instances where any tag is used. It's already inside a Javadoc. Not really how to fix this. Could you help me with this?

**Comment by aritrasinha108** [comment]
Created a PR in case you want to have a look on the changes
